GA-worksheet.md

```diff
@@ -6,18 +6,18 @@
 
 
 ##Project #2 Part #1: Header HTML
-1. What is the purpose of the header tag?
+1. What is the purpose of the header tag? A header tag is made for headers and its similar to a body, however it is meant specifically for a website's header.
 
 2. What is a Navigation Bar?
+Navigation bar is just a list of link.
+3. What tag do we use to make a navigation bar? <nav>
 
-3. What tag do we use to make a navigation bar? <      >
+4.  <ul> Stands for unordered list
+    <li> Stands for list items
 
-4.  <ul> Stands for _____________________
-    <li> Stands for ______________________
+5. What does the <a> tag do? The <a> is an anchor list, which makes list not just a text but links to another page.
```

Comment:
The li tag is separate from the a tag, so they aren't always lists. They just set up a link to go somewhere, with the destination specified in the attribute href.